generator/feature_builder.cpp

```diff
@@ -270,6 +270,24 @@ bool FeatureBuilder::PreSerialize()
     return false;
   }
 
+  // Stats shows that 1706197 POIs out of 2258011 have name == brand.
+  // Can remove duplicates, since we use "brand" only in search.
```

Comment:
But you made it displayed in PP in f7db63b959aad49840a2ab145518aed729f66bfb, no?

Reply:
Yes, we remove duplicates. Don't store a brand if it is equal to a name.